apps/dashboard/src/components/auth/inbox-playground.tsx

```diff
@@ -19,28 +20,11 @@ import { InboxPreviewContent } from './inbox-preview-content';
 
 const PLAYGROUND_CONFIG = {
   title: 'The <Inbox/> your app deserves',
-  description: "This is what your users will see. Try sending a message. We've prefilled it for you.",
+  description: 'Try sending a notification',
```

Comment:
This is a little but too generic, let's provide here some meanigful context, or setup the ground for what's to come